shared/templates/accounts_password/ansible.template

```diff
@@ -5,7 +5,31 @@
 # disruption = low
 - (xccdf-var var_password_pam_{{{ VARIABLE }}})
 
-- name: Ensure PAM variable {{{ VARIABLE }}} is set accordingly
+{{% if product == "ol8" %}}
+- name: {{{ rule_id }}} - Find pwquality.conf.d files
+  find:
```

Comment:
Could you use the complete name of the module, please? I mean including the collection name. Valid for the other modules too.

Reply:
For example: `ansible.builtin.find` instead of `find`.